src/lib/select/select.ts

```diff
@@ -470,6 +481,22 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
   ngOnInit() {
     this._selectionModel = new SelectionModel<MatOption>(this.multiple, undefined, false);
     this.stateChanges.next();
+
+    // We need `distinctUntilChanged` here, because some browsers will
+    // fire the animation end event twice for the same animation.
```

Comment:
Is there an issue for this on angular/angular?

Reply:
I couldn't find one so I filed https://github.com/angular/angular/issues/24084. I'll add a link to the issue in a comment.